app/views/users/new.haml

```diff
@@ -1,57 +1,54 @@
 = form_for @user, url: users_path, method: :post do |f|
-  %table#signup.form-table
-    %thead
-      %tr
-        %th.table-title{colspan: 2} Sign Up
-    %tbody
-      %tr#signup-username
-        %th.sub.vtop= f.label :username
-        %td.even
-          = f.text_field :username, placeholder: "Username", autofocus: true, autocomplete: "username", minlength: User::MIN_USERNAME_LEN, maxlength: User::MAX_USERNAME_LEN, required: true
-          .user-alert.hidden
-            = image_tag 'icons/exclamation.png', alt: '!', title: '', class: 'vmid'
```

Comment:
![image](https://github.com/user-attachments/assets/e3fc9dad-5cda-40d5-b268-65903fdd1095)
![image](https://github.com/user-attachments/assets/2e878855-1e31-44c8-8afb-3236d88e5e92)

Without the `.vmid` the icon isn't aligned with the text.

Reply:
Made a PR into this branch to fix this https://github.com/glowfic-constellation/glowfic/pull/2383